csm/core/services/session_factory.py

```diff
@@ -0,0 +1,166 @@
+# CORTX-CSM: CORTX Management web and CLI interface.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+from typing import Optional
+from cortx.utils.data.db.db_provider import DataBaseProvider
+from csm.core.blogic import const
+from cortx.utils.data.access import Query
+from cortx.utils.data.access.filters import Compare
+from csm.core.services.permissions import PermissionSet
+from csm.core.blogic.models import CsmModel
+from datetime import datetime
+from csm.core.data.models.session import (SessionModel, SessionCredentialsModel, 
+                                          PermissionSetModel)
+
+class SessionCredentials:
```

Comment:
Rename csm/core/services/session_factory.py --> csm/core/services/session/session_factory.py

Reply:
moved to csm/core/services/session/session_factory.py